src/Relude/Extra/Bifunctor.hs

```diff
@@ -25,7 +25,9 @@ module Relude.Extra.Bifunctor
        , secondF
        ) where
 
-import Relude
+import Relude.Functor (Bifunctor, Functor, second, fmap, first)
+import Relude.Function ((.))
+import Relude.Extra.Validation (bimap)
```

Comment:
We've been using implicit `import Relude` in extra modules because `Relude` doesn't reexport extra modules. So it's fine to save some time for us here and don't specify each module explicitly.

Reply:
It's not as easy possible as there are many cycles - `Relude` imports `Relude.Extra` which imports `Relude.Extra.X` which imports `Relude`. I'll try :)

Reply:
But `Relude` mustn't export `Relude.Extra`. And it currently doesn't do this:

* https://github.com/kowainik/relude/blob/master/src/Relude.hs

The main motivation behind having `Relude.Extra`  modules is that they are not exported by default from `Relude`.

Reply:
Thank you, now it is building and tests are running through.
I've needed to remove `{-# LANGUAGE Safe #-}` from `Extra.hs` otherwise I got errors:

```
src/Relude/Extra.hs:31:1: error:
    Relude.Extra.Group: Can't be safely imported!
    The module itself isn't safe.
   |
31 | import Relude.Extra.Group
   | ^^^^^^^^^^^^^^^^^^^^^^^^^

src/Relude/Extra.hs:33:1: error:
    Relude.Extra.Map: Can't be safely imported!
    The module itself isn't safe.
   |
33 | import Relude.Extra.Map
```